cdk/README.md

```diff
@@ -1,48 +1,17 @@
-# CDK migration
+# CDK development
 
-This holds code for the issue https://github.com/superwerker/superwerker/issues/328
-
-# 1. CDK bootstrap
```

Comment:
Maybe add link to [cdk/superwerker-distribution/README.md](https://github.com/superwerker/superwerker/pull/449/files#diff-12507e93ca96faad96620552130e966e79fdb6c22b9b73f4b0972cba23c82815)